dftimewolf/lib/collectors/grr_hunt.py

```diff
@@ -54,8 +97,44 @@ def _CreateHunt(
     """
     runner_args = self.grr_api.types.CreateHuntRunnerArgs()
     runner_args.description = self.reason
+
+    if self.match_mode:
+      if self.match_mode.lower() == 'any':
```

Comment:
We can `lower` the match mode in `HuntSetup()` and assume it's already formatted as expected here.

Reply:
Done.